elastic/kibana
# [infra] hosts table metrics

## Summary

Closes https://github.com/elastic/kibana/issues/172475

- record performance event of the host table data loading
- infra synthtrace now installs `system` package and can produce all datasets queried by host view
- [lens viz](https://telemetry-v2-staging.elastic.dev/s/kibana-performance/app/visualize#/edit/f88d3180-9f10-11ee-b4ef-7533a045abfc?_g=h@97e8101&_a=h@a8f21d5) (internal)

### Testing
- run journey with `PERFORMANCE_ENABLE_TELEMETRY=1 node scripts/run_performance.js -v --journey-path x-pack/performance/journeys/infra_hosts_view.ts`
- check data is ingested in lens
